_includes/mobile/intro-bidding-only.md

```diff
@@ -0,0 +1,115 @@
+## Overview
+
+This how-to guide covers the original approach for integrating the Prebid SDK into your app with the GMA SDK. It utilizes:
+
+- **Prebid SDK** and **Prebid server** to handle the bidding and auction process.
+- **GAM** and the **Google Mobile Ads (GMA) SDK** manage the ad inventory and select the winning ad to display.
+- **Prebid Universal Creative** renders display ads when a Prebid bid wins.
+- **GMA SDK** renders video ads when a Prebid bid wins.
```

Comment:
Two points here
- remove `video` ads, since **GMA SDK**  is used for both banner and video. 
- add Google IMA SDK for In-stream integration scenario. In-Stream don't use GMA SDK at all.